source/core-concepts/notebooks.rst

```diff
@@ -0,0 +1,11 @@
+.. meta::
+    :description: What are Valohai hosted notebooks? Explore your data and model in ephemeral environments.
+
+Notebooks
+=========
+
+**A notebook** is a hosted, short-lived Jupyter environment for experimenting with your model.
```

Comment:
I think this is good as it describes how the Valohai hosted notebooks work. Just for clarification, I'd repeat that they are Valohai hosted notebook vs Jupyter notebooks, as the naming might confuse people.

What about for example:
`**A notebook** in Valohai is a hosted, ...`

Another thing we could maybe do, is add it the clarification below too:
`...seamlessly run your Jupyter notebook contents...` (instead of just notebook)

Reply:
fixed